server/middleware/checkWhetherToShowServiceOnboardingBanner.ts

```diff
@@ -15,7 +15,7 @@ const checkWhetherToShowServiceOnboardingBanner = async (req: Request, res: Resp
   )
 
   res.locals.showServiceOnboardingBanner =
-    !userRoles.includes(ApplicationRoles.ROLE_EDUCATION_WORK_PLAN_EDITOR) &&
+    !userRoles.includes(ApplicationRole.ROLE_EDUCATION_WORK_PLAN_EDITOR) &&
```

Comment:
This will need to change with the move away from this role name, but it is not the subject of this PR